src/Utility/ExprFunctions.h

```diff
@@ -33,9 +33,12 @@ namespace ExprEval {
 
 class EvalFunc : public ScalarFunc
 {
-  std::vector<ExprEval::Expression*> expr; //!< Roots of the expression tree
-  std::vector<ExprEval::FunctionList*>  f; //!< Lists of functions
-  std::vector<ExprEval::ValueList*>     v; //!< Lists of variables and constants
+  using Expression = ExprEval::Expression<Real>;     //!< Type alias for expression tree
+  using FunctionList = ExprEval::FunctionList<Real>; //!< Type alias for function list
+  using ValueList = ExprEval::ValueList<Real>;       //!< Type alias for value list
```

Comment:
Consider moving these three definitions out into global scope such that you don't have to repeat them for both classes?

Reply:
It will make more sense once EvalFunc / EvalFunctions are templated over a scalar type in upcoming commits. Moving to global scope would destroy their purpose.